<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>artColorEffectDialog</class>
 <widget class="QDialog" name="artColorEffectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Effect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Effect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxEffect">
       <item>
        <property name="text">
         <string>Sepia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vivid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RGB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Heat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Radium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red-green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monochrome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Old photo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Night vision</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>X-ray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shades</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XPRO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow-blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow-bluish</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chrome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Esses</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="toolboxLayout"/>
   </item>
   <item>
    <widget class="ADM_QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ADM_QSlider</class>
   <extends>QSlider</extends>
   <header>ADM_toolkitQt.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>artColorEffectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>artColorEffectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
